extensions/amp-horizontal-scroller/0.1/amp-horizontal-scroller.css

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software'
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-horizontal-scroller {
+}
+.-amp-horizontal-scroller {
+  overflow-x: auto !important;
```

Comment:
@darobin i don't think that is the right place to bring this up, why don't you file an issue on amp-carousel to have it support the container layout instead. That would probably help us get to a good discussion/solution.